components/maincontroller.js

```diff
@@ -273,6 +275,7 @@
             if (!isPlaying()) {
                 embyActions.displayUserInfo($scope, data.serverAddress, data.accessToken, data.userId);
             }
+            embyActions.reportPlaybackProgress($scope, getReportingParams($scope), true, "playbackstop");
```

Comment:
Could you add a comment here with the explanation? Might be useful in the future.

Reply:
yeah makes sense, done.